bitnami/minio/templates/api-ingress.yaml

```diff
@@ -9,18 +9,11 @@ kind: Ingress
 metadata:
   name: {{ include "common.names.fullname" . }}-api
   namespace: {{ .Release.Namespace | quote }}
-  labels: {{- include "common.labels.standard" . | nindent 4 }}
-    {{- if .Values.commonLabels }}
-    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
-    {{- end }}
-  annotations:
-    {{- $annotations := fromYaml (include "minio.apiIngress.annotations" .) -}}
-    {{- if $annotations }}
-    {{- include "common.tplvalues.render" (dict "value" $annotations "context" $) | nindent 4 }}
-    {{- end }}
-    {{- if .Values.commonAnnotations }}
-    {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
-    {{- end }}
+  labels: {{- include "common.labels.standard" ( dict "customLabels" .Values.commonLabels "context" $ ) | nindent 4 }}
+  {{- $annotations := merge (fromYaml (include "minio.apiIngress.annotations" .)) .Values.commonAnnotations }}
```

Comment:
I'd rather use a pipe like we normally do in similar situations

```
include "minio.apiIngress.annotations" . | fromYaml
```

Reply:
fixed at https://github.com/bitnami/charts/pull/18349/commits/7af78dce4f2e395f4ceaacb168f81b1c4b8ca635